atom/browser/ui/file_dialog_mac.mm

```diff
@@ -144,10 +158,13 @@ void ShowOpenDialog(atom::NativeWindow* parent_window,
                     const base::FilePath& default_path,
                     const Filters& filters,
                     int properties,
+                    const std::string& message,
                     const OpenDialogCallback& c) {
   NSOpenPanel* dialog = [NSOpenPanel openPanel];
 
-  SetupDialog(dialog, title, button_label, default_path, filters);
+  SetupDialog(dialog, title, button_label, default_path, filters, message,
+          // NSOpenPanel does not support name_field_label and shows_tag_field
+          "", false);
```

Comment:
The alignment here appear off, can you align this with the `dialog` argument on line 165?